### ISSUE TITLE
Feature / Bulk topics deletion

### ISSUE THREAD

**Issue by xakassi** [tchiotludo/akhq#542]
Hi, @tchiotludo !

It would be nice to have an ability to bulk delete topics.

I think from UI side it should be check-boxes near each topic. And "Select all" button and "Delete selected" button at the top or at the bottom.

Also it would be nice if we can filter topics by name, then "Select all" and remove them. So it will be deletion by name. Seems very convenient for dev purposes.

**Comment by tchiotludo** [comment]
Thanks @xakassi, for now, we will not handled multiple page on this features, only the current page will be deleted. 
Multipage is more complicated I think

**Comment by xakassi** [comment]
Yes, I agree. I think, multipage deletion is quite unobvious thing. I suppose, user should see what he is going to delete (since it's a UI).
